docs/protocol/substrate-cli.mdx

```diff
@@ -371,6 +371,8 @@ We will be downloading two files for your respective operating system.
   2022-02-03 10:52:29 ⚙️  Syncing, target=#215883 (2 peers), best: #55 (0xafc7…bccf), finalized #0 (0x6ada…0d38), ⬇ 850.1kiB/s ⬆ 1.5kiB/s
   ```
   5. We will then open another terminal, change to the downloads directory, then start the farmer node with the following command:
+  :::note Farming while Plotting changes
```

Comment:
I'm not sure what's going on here? I can't see this change in the preview?

Reply:
Fixed with commit 00d53e2